Makefile

```diff
@@ -0,0 +1,15 @@
+MOCHA_TARGET=test/specs.js
```

Comment:
please remove Makefile
use npm scripts
https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/